ktesting/example_test.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ktesting_test
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"k8s.io/klog/v2/ktesting"
+)
+
+func ExampleUnderlier() {
+	logger := ktesting.NewLogger(ktesting.NopTL{}, ktesting.NewConfig())
+
+	logger.Error(errors.New("failure"), "I failed", "what", "something")
+	logger.WithValues("request", 42).WithValues("anotherValue", "fish").Info("hello world")
+	logger.WithName("example").Info("with name")
+	logger.V(1).Info("higher verbosity")
```

Comment:
`logger.V(6).Info("This should not be captured in the buffer")`

Reply:
extra line to go with above comment.

Reply:
Done, just with a different wording.